Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Substance/readme.md

```diff
@@ -0,0 +1,80 @@
+----
+
+Copyright (c) Contributors to the Open 3D Engine Project.For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+----
+
+# O3DE DCCsi and Substance Designer
+
+This document contains instructions and other information related to using Adobe Substance Designer with Open 3D Engine (O3DE).  O3DE has a Gem called the DccScriptingInterface (DCCsi) that helps user manage the usage of many popular Digital Content Creation tools (DCC) such as Substance.  These are primarily workflow integrations with the intent to provide an improved ease-of-use experience when these tools are used to author source content for O3DE.  The DCCsi helps with aspects such as, configuration and settings, launching tools and boostrapping them with O3DE extensions (generally python scripts using the DCC tools APIs to automate tasks)
+
+## Status:
+
+- Prototype
+
+## Revisions:
+
+- This is the first working Proof of Concept
+
+## Notes:
+
+- Currently only the latest version of Substance Designer installed via Adobe CC has been tested: 12.1.1 build 5825 commit 9adecf0b Release (2022-05-31).
+- This should work with other modern Adobe versions in the same Adobe install location:
+  - C:\Program Files\Adobe\Adobe Substance 3D Designer\Adobe Substance 3D Designer.exe
+- See the Appendix if you are trying to work with older legacy versions.
+
+## Setup
+
+### General Info
+
+The DCCsi utilizes a set of package dependancies defined in a requirements.txt file
+
+[< o3de> / Gems / AtomLyIntegration / TechnicalArt / DccScriptingInterface / requirements.txt](https://github.com/o3de/o3de/blob/development/Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/requirements.txt)
+
+When the O3DE engine is built (cmake), these packages are installed into the O3DE python interpretter/runtime, so python scripts utilizing them have access to these packages; i.e. when a python script is executed by the main Editor.exe python runtime.
+
+Many DCC applications such as Substance Designer also come with a managed python interpretter:
+
+- O3DE doesn't use a user or system installed python interpretter, nor do most DCC apps we are aware of.
+
+- The DCC apps python version may be different then O3DE e.g. python 3.7.12 vs 3.9.9.
+
+- O3DE and most DCC apps we are aware of don't make use of virtual enviornments, which is a common way to manage package dependancies for different versions of python.
```

Comment:
"dependancies" -> "dependencies"

Reply:
will fix when I finish and then update the readme doc